client/src/components/categorical/index.js

```diff
@@ -182,6 +182,35 @@ class Categories extends React.Component {
           }
         />
 
+        {writableCategoriesEnabled ? (
```

Comment:
this change seems unrelated to gene sets, and just moves the Create button for categorical annotation to the top.  The layout for the tools is also now quite odd (see snapshot below).   So I'm wondering:
a) why change it at all in this PR?
b) or if we are going to change it, due to the new Create Gene Sets button, why not clean up the layout so it is more visually pleasing?

![image](https://user-images.githubusercontent.com/3740062/111001749-2f028180-8339-11eb-9d69-9dcbdc4b8195.png)

Reply:
Great question. Yes, I want to leave create category at the top, since there are now two primary actions that might be buried by a large list of categories and/or genesets. The area above `create new gene set` is still in flight. This is captured in #2059

![image](https://user-images.githubusercontent.com/1770265/111002116-db9d2d00-8352-11eb-89c4-e059a6edbba7.png)